lib/msal-core/src/AuthResponse.ts

```diff
@@ -12,9 +12,9 @@ export type AuthResponse = {
     tenantId: string;
     tokenType: string;
     idToken: IdToken;
-    accessToken: object;
+    accessToken: string;
     scopes: Array<string>;
-    expiresIn: string;
+    expiresIn: Date;
```

Comment:
if we are calling expiresIn, it should have the relative time period (e.g. 300 secs)
if you return date object, then we should name it different

Reply:
should just be a number type for milliseconds

Reply:
Trying to be consistent with other MSAL platforms here. MSAL Android and MSAL Obj-C are using the corresponding Date format for their platforms, and returning a var called 'expiresOn'. I will change this var here to be expiresOn instead of expiresIn to better reflect its usage.